src/commands/deploy.js

```diff
@@ -248,38 +49,45 @@ async function parseDeploymentOptions(cargo, config = {}, { adminAddress, confir
   }
 }
 
-async function deploy(archwayd, { build = true, verify = true, dryRun = false, ...options } = {}) {
+/**
+ * @see Build
+ * @see Store
+ * @see Instantiate
+ * @deprecated since v1.2.0
+ */
+async function deploy(archwayd, { build = true, dryRun = false, ...options } = {}) {
+  build && await Build({ optimize: true });
+
   // TODO: call archwayd operations with the --dry-run flag
   if (dryRun) {
-    console.info('Building wasm binary...\n');
-    await new ScriptRunner().run('wasm');
```

Comment:
is this expected? just an empty `if` with return?

Should we print some meaningful message here?

Reply:
This is expected to keep backward compatibility when running the `deploy` command with the `--dry-run` flag.